core/spec/models/spree/wallet/add_payment_sources_to_wallet_spec.rb

```diff
@@ -13,5 +14,27 @@
         order.user.wallet.wallet_payment_sources.count
       }.by(1)
     end
+
+    context "when the default wallet payment source is used and more a recent wallet payment exists" do
+      before do
+        credit_card_one = user.wallet.add(create(:credit_card, user: user))
+        credit_cart_two = user.wallet.add(create(:credit_card, user: user))
+        user.wallet.default_wallet_payment_source = credit_card_one # must be the first created card
+        order.payments.first.update!(source: credit_card_one.payment_source)
+        flunk unless credit_card_one.payment_source.reusable?
```

Comment:
Can't we make sure that the payment_source is reusable in some other way?

Reply:
Hmm, for context, the reason I added this is because if the credit card was not reusable, the spec could become a false positive. Now, instead, I added `expect(user.wallet).to receive(:add).and_call_original` to the specs, WDYT?

Reply:
I see. My point is, is there a way to ensure that through the factory itself?

Reply:
Not directly:
https://github.com/solidusio/solidus/blob/a5c2e41e22441cf3fe648c21bf438c6d92c4130d/core/app/models/spree/credit_card.rb#L123-L131

Seeing as `reusable?` is already returning `true`. I think it dilutes the spec a lot of we add unnecessary setup to make sure `reusable?` is explicitly `true`. Is `expect(user.wallet).to receive(:add).and_call_original` not a suitable alternative to ensure we're not getting false positives?